presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -169,19 +172,57 @@ See also :doc:`/language/timestamp`
 
     Example: ``TIME '01:02:03.456 America/Los_Angeles'``
 
+.. _timestamp-data-type:
+
 ``TIMESTAMP``
 ^^^^^^^^^^^^^
 
     Instant in time that includes the date and time of day without a time zone.
```

Comment:
Remove the duplicate description. This section should only mention that this is an alias.

Reply:
See comment